src/utils/frame.js

```diff
@@ -160,6 +171,35 @@ export function annotateFrame(frame: Frame) {
   return frame;
 }
 
+function annotateBabelAsyncFrames(frames: Frame[]) {
+  const firstIndex = findIndex(
```

Comment:
I can't quite tell how the grouping here is done. Is the approach of firstIndex and lastIndex going to have problems if one async function is called inside of another one, so you potentially have multiple `tryCatch`s in the frame list?

Reply:
@calebstdenis do you know how to test this with babel...
The main thing is to add this to a file and then run babel to create the output file


```js
async function first() {
  debugger;
}

async function second() {
  await first()
}

second
```

Reply:
@loganfsmyth You're right. Thanks for catching this!
![image](https://user-images.githubusercontent.com/7321311/36800857-8562c8a2-1c7e-11e8-89ba-b147c64c8510.png)

Reply:
@loganfsmyth Commit 1fac0ea980c493e063df60729d441f23ac9bd2e9 should address this. Also, the code is hopefully more self-explanatory now. 

![image](https://user-images.githubusercontent.com/7321311/36925581-890136e6-1e41-11e8-8df3-e10b4eda9142.png)